### ISSUE TITLE
[Suggestion] Emulator testing squad.

### ISSUE THREAD

**Issue by INFINITY99K** [Ryujinx/Ryujinx#2858]
When I scroll through the issues segment there are a lot of repeated issues for example Super Mario Party Superstars has more than 2 entries on the list so, if a so called "squad" was formed to test all the games that are going to be released in the future then, they report the bugs to you.
And as for the older games they can split themselves to two teams one to test the newly released titles and one to test out older titles and report if they have any inconsistencies or, graphical glitches.
And as for the UI and stability another team will test the games and compare them between different versions of the emulator to see if there is any regression and will test the UI to see if there is some instability in it.
So basically three teams: one for bug tracking in newly released games, one for older titles, and a one for general stability and consistency.

And that will make:
A: Tracking bugs will be much easier.
B: Fixes will come in a rapid rate.
C: The progress and development of the emulator will be much more mature.
D: Won't be overwhelmed by the amount of open issues.

I don't suggest that you completely ignore the issues section as it'll make a room to track issues of more obscure games that don't get marketed that well and somebody in the userbase will play it.
